src/models.ts

```diff
@@ -471,10 +496,11 @@ export class Sequential extends Model {
    *   layers: [tf.layers.dense({units: 1, inputShape: [10]})]
    * });
    * model.compile({optimizer: 'sgd', loss: 'meanSquaredError'});
-   * const history = await model.fit(tf.ones([8, 10]), tf.ones([8, 1]), {
+   * const h = await model.fit(tf.ones([8, 10]), tf.ones([8, 1]), {
```

Comment:
Why 'h'?  seems like a bad variable choice name, and if its to deal with the h.history.loss in the lines later, it makes we think we have the wrong return type on fit?